modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -409,7 +409,7 @@ label:enterprise-only[]
 
 [WARNING]
 ====
-Incremental import is not supported for `block` format yet.
+Incremental import does not support `block` format.
```

Comment:
This functionality is currently being built. We should not forget to remove this note once it is supported